src/Command/I18nExtractCommand.php

```
@@ -692,6 +692,9 @@ protected function checkUnchanged(string $oldFile, int $headerLength, string $ne
             return false;
         }
         $oldFileContent = file_get_contents($oldFile);
+        if ($oldFileContent === false) {
+            return false;
```

Could we throw an exception here? The file being broken seems important.